cmd/zoekt-sourcegraph-indexserver/queue.go

```diff
@@ -379,7 +440,7 @@ func (pq *pqueue) Pop() interface{} {
 	return item
 }
 
-// lessQueueItemPriority returns true if the indexing priority x is less than that of y.
+// lessQueueItemPriority returns true if indexing x should be prioritized over indexing y
```

Comment:
nit: clarify comment to distinguish less in sort order (ascending) means priority is greater than, not less than

Reply:
we could also rename method to `lessQueueItem` or `greaterPriority`, but the comment itself should be sufficient to see how it's properly used by `Less()`.

Reply:
Yeah I think the previous comment was still correct in the context of the heap and the context of being called by Less() which we need to implement. Updated comment is just in case we ever call this for comparison without the context of the min heap and it simply gives us "greater priority" (no reference to "Less than" in the heap sort context)